deploy/nginx_example.conf

```diff
@@ -4,6 +4,10 @@ upstream satellity_http_server {
 
 server {
   listen 80;
+
+  access_log /var/log/nginx/access.log;
+  error_log /var/log/nginx/error.log;
```

Comment:
Nginx has default log config, I don't think we need this

Reply:
ok, I will confirm